Overview.bs

```diff
@@ -479,7 +479,11 @@ The algorithm:
      produce the <var>target URI</var>.
 
 2.  Retrieve the contents of <var>target URI</var> using the fetching capabilities of the implementing user agent. For web browsers,
-     [[fetch]] should be used. Return the retrieved contents as <var>patch file</var>, or an error if the fetch resulted in an error.
+     [[fetch]] should be used. When using [[fetch]] the requests for patches should use the same CORS settings as the IFT font which
```

Comment:
This might be a bit easier to parse: "When using [[fetch]] a request for patches should use the same CORS settings as the initial request for the IFT font. This means that for a font loaded via CSS the patch quest would follow:"